dash.js

```diff
@@ -907,6 +889,7 @@ var MyDash = GObject.registerClass({
     }
 
     _initializeIconSize(max_size) {
+        log("initializing icon size...." + max_size);
```

Comment:
Guess we can drop this at merge time

Reply:
Yes :laughing: this was... tricky to debug.